src/imagej/util.py

```diff
@@ -0,0 +1,105 @@
+from itertools import cycle
+from shutil import get_terminal_size
+from threading import Thread
+from time import sleep
+from typing import List
+
+
+class Loader:
+    def __init__(
+        self, start_msg="Loading...", end_msg="Done!", timeout=0.1, style="rotate"
+    ):
```

Comment:
We should probably move this upstream to ScyJava, no? There is nothing image-specific about this code.